### PR TITLE
update to android studio sample

### PR DESCRIPTION
**Pull request by oalami**
@lesv please review and merge

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 22
+    buildToolsVersion "22.0.1"
+
+    defaultConfig {
+        applicationId "com.google.devrel.samples.ttt"
+        minSdkVersion 14
```
Comment (lesv) [inline app/build.gradle]: These days you should be build 17 as min. (Lowest Android Team recommends supporting)

File: app/build/generated/source/buildConfig/androidTest/debug/com/google/devrel/samples/ttt/test/BuildConfig.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
```
Comment (lesv) [inline app/build/generated/source/buildConfig/androidTest/debug/com/google/devrel/samples/ttt/test/BuildConfig.java]: Not sure build/generated should be checked in (et al)

**Comment by lesv** [comment]
PTAL

**Comment by oalami** [comment]
@lesv ugh, shouldn't try to send a PR right before dinner. Removed the build files + updated the min version, back to you.

**Comment by lesv** [comment]
I also sent you a suggested .gitignore

On Wed, Mar 25, 2015 at 9:13 PM, Ossama Alami <notifications@github.com>
wrote:

> @lesv <https://github.com/lesv> ugh, shouldn't try to send a PR right
> before dinner. Removed the build files + updated the min version, back to
> you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86332228>
> .
>



-- 
Les Vogel | Cloud Developer Relations | lesv@google.com | 408-676-7023

**Comment by oalami** [comment]
updated.

On Wed, Mar 25, 2015 at 9:21 PM, Les Vogel <notifications@github.com> wrote:

> I also sent you a suggested .gitignore
>
> On Wed, Mar 25, 2015 at 9:13 PM, Ossama Alami <notifications@github.com>
> wrote:
>
> > @lesv <https://github.com/lesv> ugh, shouldn't try to send a PR right
> > before dinner. Removed the build files + updated the min version, back to
> > you.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86332228
> >
> > .
> >
>
>
>
> --
> Les Vogel | Cloud Developer Relations | lesv@google.com | 408-676-7023
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appengine-endpoints-tictactoe-android/pull/6#issuecomment-86336102>
> .
>



-- 
Ossama Alami | Google Developer Relations | oa@google.com

**Comment by lesv** [comment]
LGTM - If you had merged the repos (Android & GAE), you wouldn't need to checkin the generated code.  They would just generate it when needed.  We had Antoine also merge the iOS so that folks could look just one place and get the right thing.  That said, LGTM, if you feel inspired great, if not, just merge, or ask again and I can.

**Comment by oalami** [comment]
doesn't look as I have write access. Will figure out why tomorrow, in the meantime, please merge.

**Pull request merged**